guides/v2.3/graphql/product/product-interface.md

```diff
@@ -68,15 +70,46 @@ Attribute | Data type | Description
 
 ### ProductPrices object {#ProductPrices}
 
+{:.bs-callout-info}
+The `ProductPrices` object has been deprecated. Use the `PriceRange` object instead.
+
 The `ProductPrices` object contains the regular price of an item, as well as its minimum and maximum prices. Only composite products, which include bundle, configurable, and grouped products, can contain a minimum and maximum price.
 
 Attribute |  Data Type | Description
 --- | --- | ---
-`maximalPrice` | Price | Used for composite (bundle, configurable, grouped) products. This is the highest possible final price for all the options defined within a composite product. If you're specifying a price range, this would be the "to" value.
-`minimalPrice` | Price | Used for composite (bundle, configurable, grouped) products. This is the lowest possible final price for all the options defined within a composite product. If you're specifying a price range, this would be the "from" value.
-`regularPrice` | Price | The base price of a product.
+`maximalPrice` | Price | Deprecated. Use `PriceRange.maximum_price` instead. Used for composite (bundle, configurable, grouped) products. This is the highest possible final price for all the options defined within a composite product. If you're specifying a price range, this would be the "to" value.
+`minimalPrice` | Price | Deprecated. Use `PriceRange.minimum_price` instead. Used for composite (bundle, configurable, grouped) products. This is the lowest possible final price for all the options defined within a composite product. If you're specifying a price range, this would be the "from" value.
+`regularPrice` | Price | Deprecated. Use `PriceRange.maximum_price` or `PriceRange.minimum_price` instead. The base price of a product.
+
+### PriceRange object {#PriceRange}
+
+The `PriceRange` object defines the price range for a product. If a product only has a single price, the minimum and maximum price will be the same.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`maximum_price` | ProductPrice | The highest possible final price for a product
+`minimum_price` | ProductPrice | The lowest possible final price for a product
+
+### ProductPrice object {#ProductPrice}
+
+The `ProductPrice` object includes the regular price, final price, and any discounts applied to a product.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`discount` | ProductDiscount | The amount of the discount applied to the product. It represents the difference between the `final_price` and `regular_price`
+`final_price`| Money! | The price of the product after applying discounts
+`regular_price` | Money! | The regular price of the product, without any applied discounts
 
-#### Price object {#Price}
+### ProductDiscount object {#ProductDiscount}
+
+The `ProductDiscount` object indicates whether the discount applied to a product is fixed amount, such as $5, or a percentage, such as 10%. The discount originates from special pricing or a catalog price rule.
```

Comment:
"...indicates whether the discount applied..." It doesn't show "whether" its fixed amount or percent, it shows the price discounts expressed as a percent and amount (again, I'm not great at the wording) :)